From e6b303444988717c725a71db7b21417839321463 Mon Sep 17 00:00:00 2001
From: Yi Zhao <yi.zhao@windriver.com>
Date: Tue, 4 Aug 2020 16:48:12 +0800
Subject: [PATCH] fc/sysnetwork: update file context for ifconfig

The ifconfig was moved from sbin to bin with oe-core commit:
c9caff40ff61c08e24a84922f8d7c8e9cdf8883e. Update the file context for
it.

Upstream-Status: Inappropriate [embedded specific]

Signed-off-by: Yi Zhao <yi.zhao@windriver.com>
---
 policy/modules/system/sysnetwork.fc | 1 +
 1 file changed, 1 insertion(+)

diff --git a/policy/modules/system/sysnetwork.fc b/policy/modules/system/sysnetwork.fc
index d8902d725..9ec4eefb7 100644
--- a/policy/modules/system/sysnetwork.fc
+++ b/policy/modules/system/sysnetwork.fc
@@ -43,6 +43,7 @@ ifdef(`distro_redhat',`
 /usr/bin/dhcpcd		        --	gen_context(system_u:object_r:dhcpc_exec_t,s0)
 /usr/bin/ethtool		    --	gen_context(system_u:object_r:ifconfig_exec_t,s0)
 /usr/bin/ifconfig		    --	gen_context(system_u:object_r:ifconfig_exec_t,s0)
+/usr/bin/ifconfig\.net-tools		--	gen_context(system_u:object_r:ifconfig_exec_t,s0)
 /usr/bin/ip			        --	gen_context(system_u:object_r:ifconfig_exec_t,s0)
 /usr/bin/ipx_configure		--	gen_context(system_u:object_r:ifconfig_exec_t,s0)
 /usr/bin/ipx_interface		--	gen_context(system_u:object_r:ifconfig_exec_t,s0)
-- 
2.17.1

